recipes/hdf5/build.sh

```diff
@@ -2,8 +2,7 @@
 
 ./configure --prefix=$PREFIX \
     --enable-linux-lfs --with-zlib \
-    --with-pthread=yes  --enable-cxx --with-default-plugindir=$PREFIX/lib/hdf5/plugin \
-    --disable-silent-rules  # To make Travis happy with the level of activity.
+    --with-pthread=yes  --enable-cxx --with-default-plugindir=$PREFIX/lib/hdf5/plugin
```

Comment:
However, the build still ended up timing out for ~10mins. So, there is something else going on. As it was the `cache` test, expect this to be memory related. Perhaps, it is going to swap, which would explain why this bogs down.